room-kit/src/main/java/live/hms/roomkit/ui/meeting/PreviewFragment.kt

```diff
@@ -243,15 +243,23 @@ class PreviewFragment : Fragment() {
         // don't allow editing the name if there's supposed to be a fixed one
 
         var introAnimationOffset = 450
-        if (meetingViewModel.getHmsRoomLayout()
-                ?.getPreviewLayout(roleName)?.default?.elements?.previewHeader?.title.isNullOrEmpty()
-        ) {
-            binding.nameTv.visibility = View.GONE
-        } else {
+        if (!meetingViewModel.liveClassName.isNullOrEmpty()){
+            binding.nameTv.text = meetingViewModel.liveClassName
```

Comment:
we need to change it like 
binding.nameTv.text = "Welcome to " + meetingViewModel.liveClassName
right ? @Aswinks-entri